### PR TITLE
Docs docker 2 - small polish tweak

### PR DESCRIPTION
**Pull request by gregelin**
Add a clarifying comment identifying the database as the default database.

### CODE REVIEW

File: docs/source/deploy_docker.md [hunk 1]
```diff
@@ -8,52 +8,55 @@
 | Nightly Build on Docker   | [https://hub.docker.com/r/govready/govready-q-nightly/](https://hub.docker.com/r/govready/govready-q-nightly/)  |
 
 
-## Running the Docker container
+## Quickstart for GovReady-Q on Docker
 
 Make sure you first [install Docker](https://docs.docker.com/engine/installation/) and, if appropriate, [grant non-root users access to run Docker containers](https://docs.docker.com/engine/installation/linux/linux-postinstall/#manage-docker-as-a-non-root-user) (or else use `sudo` when invoking Docker below).
 
-Start the container in the background:
+Start and running container in the background (e.g. detached):
```
Comment (peterkaminski) [inline docs/source/deploy_docker.md]:
```suggestion
Start and run container in the background (i.e. detached):
```

File: docs/source/deploy_docker.md [hunk 2]
```diff
@@ -8,52 +8,55 @@
...
 
+	# Run the docker container detached mode
```
Comment (peterkaminski) [inline docs/source/deploy_docker.md]:
```suggestion
	# Run the docker container in detached mode
```

File: docs/source/deploy_docker.md [hunk 3]
```diff
@@ -8,52 +8,55 @@
...
 	docker container run --name govready-q --detach -p 8000:8000 govready/govready-q
 
-For more complex setups, using our run script instead will be easier:
+	# Create admin account and organization data
+	docker container exec -it govready-q first_run
 
-	wget https://raw.githubusercontent.com/GovReady/govready-q/master/deployment/docker/docker_container_run.sh
-	chmod +x docker_container_run.sh
-	./docker_container_run.sh
+	# Stop, start container
```
Comment (peterkaminski) [inline docs/source/deploy_docker.md]:
```suggestion
	# Stop, start container (when needed)
```

File: docs/source/deploy_docker.md [hunk 4]
```diff
@@ -8,52 +8,55 @@
...
+Your GovReady-Q site may not load immediately while GovReady-Q initializes your database for the first time. Wait for the site to become available.
+
+Because of HTTP Host header checking, you must use `localhost` to access the site or another hostname if configured using the `--address` option documented below.
 
 If the site does not come up, check the container logs for an error message:
 	
 	docker container logs govready-q
 
-With the container started and the database initialized, run our first-run script to create a Django database superuser and set up your first organization:
+The GovReady-Q default SQLite database is only persisted within the container by default. The database will persist between `docker container stop`/`docker container start` commands, but when the container is removed from Docker (i.e. using `docker container rm`) the data will be destroyed. See the _Persistent database_ section below for connecting to a database outside of the container.
```
Comment (peterkaminski) [inline docs/source/deploy_docker.md]:
```suggestion
The GovReady-Q default SQLite database is ordinarily only active within the container lifetime. The database will persist between `docker container stop`/`docker container start` commands, but when the container is removed from Docker (i.e. using `docker container rm`) the data will be destroyed. See the _Persistent database_ section below for connecting to a database outside of the container for production data. 
```

Comment (gregelin) [reply docs/source/deploy_docker.md]: resolved

**Comment by gregelin** [comment]
@peterkaminski Made a few of suggested changes and also merged in previously accepted changes from #630

**Pull request merged**